app/components/vote-overview.ts

```diff
@@ -1,6 +1,4 @@
 import Component from '@glimmer/component';
-import { tracked } from '@glimmer/tracking';
-import * as d3 from 'd3';
```

Comment:
This one has the most impact on the app. Now only the map route imports d3 so Embroider can move it into the map route bundle which means the main bundle is a little bit smaller again 🎉.